src/main/java/net/dries007/tfc/util/Helpers.java

```diff
@@ -223,10 +227,86 @@ public static EnumFacing getASolidFacing(World worldIn, BlockPos pos, @Nullable
         return null;
     }
 
-    @Nonnull
+    @Nullable
```

Comment:
Don't change this `@Nonnull` - if it is null there is no point having it here, it just is unnecessary complication. The nonnull is intentional to prevent injected fields from giving null / constant conditions warnings everywhere. See discussion in discord.
```suggestion
    @Nonnull
```

Reply:
Reverted. I excluded that change from being committed now